### DISCUSSION TITLE
component for formily

### DISCUSSION THREAD

**Discussion by umi-soft** [element-plus/element-plus#988] [Ideas]
# Formily is a great product, has plan to provide offical component for formily?

**Comment by zazzaz** [comment]
Hi, you can check our reply here.

https://github.com/element-plus/element-plus/issues/983#issuecomment-744000973

**Reply by umi-soft** [reply]
I add here，in order to be tracked

**Reply by jw-foss** [reply]
Thank you for doing so!
